.github/workflows/create_release.yml

```diff
@@ -106,47 +106,54 @@ jobs:
   build-alpine:
     name: alpine
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        docker_platform: [amd64, arm64]
     steps:
     - uses: actions/setup-python@v1
       with:
         python-version: '3.x'
     - uses: actions/checkout@v1
       with:
         submodules: true
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
     - name: start docker
       run: |
-        docker run -w /src -dit --name alpine -v $PWD:/src node:lts-alpine
-        echo 'docker exec alpine "$@";' > ./alpine.sh
-        chmod +x ./alpine.sh
+        docker run -w /src -dit --platform=linux/${{ matrix.docker_platform }} --name alpine-${{ matrix.docker_platform }} -v $PWD:/src node:lts-alpine
+        echo 'docker exec alpine-${{ matrix.docker_platform }} "$@";' > ./alpine-${{ matrix.docker_platform }}.sh
+        chmod +x ./alpine-${{ matrix.docker_platform }}.sh
+
 
     - name: install packages
       run: |
-        ./alpine.sh apk update
-        ./alpine.sh apk add build-base cmake git python3 clang ninja py3-pip
+        ./alpine-${{ matrix.docker_platform }}.sh apk update
+        ./alpine-${{ matrix.docker_platform }}.sh apk add build-base cmake git python3 clang ninja py3-pip
 
     - name: install python dev dependencies
-      run: ./alpine.sh pip3 install -r requirements-dev.txt
+      run: ./alpine-${{ matrix.docker_platform }}.sh pip3 install --break-system-packages -r requirements-dev.txt
 
     - name: cmake
       run: |
-        ./alpine.sh cmake . -G Ninja -DCMAKE_CXX_FLAGS="-static" -DCMAKE_C_FLAGS="-static" -DCMAKE_BUILD_TYPE=Release -DBUILD_STATIC_LIB=ON -DCMAKE_INSTALL_PREFIX=install
+        ./alpine-${{ matrix.docker_platform }}.sh cmake . -G Ninja -DCMAKE_CXX_FLAGS="-static" -DCMAKE_C_FLAGS="-static" -DCMAKE_BUILD_TYPE=Release -DBUILD_STATIC_LIB=ON -DCMAKE_INSTALL_PREFIX=install-${{ matrix.docker_platform }}
 
     - name: build
       run: |
-        ./alpine.sh ninja install
+        ./alpine-${{ matrix.docker_platform }}.sh ninja install
 
     - name: test
-      run: ./alpine.sh python3 ./check.py
+      run: ./alpine-${{ matrix.docker_platform }}.sh python3 ./check.py
 
     - name: archive
       id: archive
       run: |
         VERSION=$GITHUB_REF_NAME
-        PKGNAME="binaryen-$VERSION-x86_64-linux"
+        ARCH=$(./alpine-${{ matrix.docker_platform }}.sh uname -m)
+        PKGNAME="binaryen-$VERSION-$ARCH-linux"
         TARBALL=$PKGNAME.tar.gz
         SHASUM=$PKGNAME.tar.gz.sha256
-        ./alpine.sh find install/ -type f -perm -u=x -exec strip {} +
-        mv install binaryen-$VERSION
+        ./alpine-${{ matrix.docker_platform }}.sh find install-${{ matrix.docker_platform }}/ -type f -perm -u=x -exec strip {} +
+        mv install-${{ matrix.docker_platform }} binaryen-$VERSION
```

Comment:
I think this directory name can also stay the same

Reply:
I did this originally because I wasn't sure if the build jobs would run in parallel on the same agent, but it doesn't look like that's the case so I've removed all this. The diff is much smaller now 👍